uint16 itis
int32 latitude
int32 longitude
st_CAVRSA[] cavrsa
